test/core/client_channel/service_config_test.cc

```diff
@@ -283,14 +296,22 @@ TEST_F(ServiceConfigTest, ValidMethodConfig) {
   const char* test_json =
       "{\"methodConfig\": [{\"name\":[{\"service\":\"TestServ\"}]}]}";
   grpc_error* error = GRPC_ERROR_NONE;
-  auto svc_cfg = ServiceConfig::Create(test_json, &error);
+  auto svc_cfg = ServiceConfig::Create(nullptr, test_json, &error);
+  EXPECT_EQ(error, GRPC_ERROR_NONE) << grpc_error_string(error);
+}
+
+TEST_F(ServiceConfigTest, ParsingDisabledViaChannelArg) {
+  const char* test_json =
+      "{\"methodConfig\": [{\"name\":[{\"service\":\"TestServ\"}]}]}";
+  grpc_error* error = GRPC_ERROR_NONE;
+  auto svc_cfg = ServiceConfig::Create(nullptr, test_json, &error);
```

Comment:
This test case seems not disabling parsing as its name `ParsingDisabledViaChannelArg`.

Reply:
I am not sure what this checks for

Reply:
Yeah, sorry, I think I started adding this, then realized I wanted to do it differently, but I forgot to remove this.

I've removed it now.